src/main/java/me/botsko/prism/actions/entity/PandaSerializer.java

```diff
@@ -0,0 +1,39 @@
+package me.botsko.prism.actions.entity;
+
+import me.botsko.prism.utils.MiscUtils;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Panda;
+
+public class PandaSerializer extends EntitySerializer {
+    protected String mainGene = null;
+    protected String hiddenGene = null;
+
+    @Override
+    protected void serializer(Entity entity) {
+        mainGene = ((Panda) entity).getMainGene().name().toLowerCase();
+        hiddenGene = ((Panda) entity).getHiddenGene().name().toLowerCase();
+    }
+
+    @Override
+    protected void deserializer(Entity entity) {
+        Panda.Gene mGene = MiscUtils.getEnum(mainGene, Panda.Gene.NORMAL);
+        Panda.Gene hGene = MiscUtils.getEnum(hiddenGene, Panda.Gene.NORMAL);
+        ((Panda) entity).setMainGene(mGene);
+        ((Panda) entity).setHiddenGene(hGene);
+}
+
+    @Override
+    protected void niceName(StringBuilder sb, int start) {
+        if (mainGene != null && hiddenGene != null) {
+            String niceName;
+            if (mainGene.equals("weak") && !hiddenGene.equals("weak")) {
+                niceName = "normal";
+            } else if (mainGene.equals("brown") && !hiddenGene.equals("brown")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5443424)